* libncurses-devel provides a libcurses symlink
* libncurses10 is now current; libncurses7 and before (without the 
hyphen in the DLL name) are old enough to ignore.

--- origsrc/cmake-2.8.1/Modules/FindCurses.cmake	2010-04-06 09:45:31.000000000 -0500
+++ src/cmake-2.8.1/Modules/FindCurses.cmake	2010-04-29 04:02:58.187387800 -0500
@@ -30,9 +30,9 @@ FIND_LIBRARY(CURSES_CURSES_LIBRARY NAMES
 FIND_LIBRARY(CURSES_NCURSES_LIBRARY NAMES ncurses )
 SET(CURSES_USE_NCURSES FALSE)
 
-IF(CURSES_NCURSES_LIBRARY  AND NOT  CURSES_CURSES_LIBRARY)
+IF(CURSES_NCURSES_LIBRARY)
   SET(CURSES_USE_NCURSES TRUE)
-ENDIF(CURSES_NCURSES_LIBRARY  AND NOT  CURSES_CURSES_LIBRARY)
+ENDIF(CURSES_NCURSES_LIBRARY)
 
 
 # Not sure the logic is correct here.
--- origsrc/cmake-2.8.1/Utilities/Release/Cygwin/CMakeLists.txt	2010-04-06 09:45:47.000000000 -0500
+++ src/cmake-2.8.1/Utilities/Release/Cygwin/CMakeLists.txt	2010-04-29 19:16:29.255401600 -0500
@@ -1,8 +1,8 @@
-FILE(GLOB INSTALLED_CURSES /usr/bin/cygncurses*.dll)
+FILE(GLOB INSTALLED_CURSES /usr/bin/cygncurses-*.dll)
 SET(MAX 0)
 FOREACH(f ${INSTALLED_CURSES})
   IF(NOT "${f}" MATCHES "\\+")
-    STRING(REGEX REPLACE ".*([0-9]).dll" "\\1" NUMBER "${f}")
+    STRING(REGEX REPLACE ".*-([0-9]*).dll" "\\1" NUMBER "${f}")
     IF(NUMBER GREATER MAX)
       SET(MAX ${NUMBER}) 
     ENDIF(NUMBER GREATER MAX)
